sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterImpl.java

```diff
@@ -323,7 +323,8 @@ public Accepted<AgentPool> beginCreateAgentPool(String agentPoolName, AgentPoolD
             () -> this.manager()
                 .serviceClient()
                 .getAgentPools()
-                .createOrUpdateWithResponseAsync(resourceGroupName(), name(), agentPoolName, agentPool.innerModel())
+                .createOrUpdateWithResponseAsync(resourceGroupName(), name(), agentPoolName, agentPool.innerModel(),
+                    null, null)
                 .block(),
```

Comment:
Make sure live test covers this call, and it passes.

Reply:
@weidongxu-microsoft 
The test cases related to `KubernetesClusterImpl.beginCreateAgentPool` are documented in `KubernetesClustersTests` and it has been passed. The test case following as : https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-containerservice/src/test/java/com/azure/resourcemanager/containerservice/KubernetesClustersTests.java#L449